pkg/synopsysctl/cmd_db_migrate.go

```diff
@@ -48,42 +45,30 @@ var dbMigrateBlackDuckCmd = &cobra.Command{
 	SilenceUsage:  true,
 	SilenceErrors: true,
 	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) == 0 {
+		if len(args) != 1 {
 			cmd.Help()
 			return fmt.Errorf("this command takes one or more arguments")
 		}
 		return nil
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-		errors := []error{}
-		for _, blackDuckName := range args {
-			blackDuckNamespace, crdNamespace, _, err := getInstanceInfo(false, util.BlackDuckCRDName, "", namespace, blackDuckName)
-			if err != nil {
-				errors = append(errors, err)
-				continue
-			}
-			log.Infof("putting Black Duck '%s' in namespace '%s' into database migration mode...", blackDuckName, blackDuckNamespace)
-
-			// Get the Black Duck
-			currBlackDuck, err := util.GetBlackduck(blackDuckClient, crdNamespace, blackDuckName, metav1.GetOptions{})
-			if err != nil {
-				errors = append(errors, fmt.Errorf("unable to get Black Duck '%s' in namespace '%s' due to %+v", blackDuckName, blackDuckNamespace, err))
-				continue
-			}
-
-			// Make changes to Spec
-			currBlackDuck.Spec.DesiredState = "DbMigrate"
-			// Update Black Duck
-			_, err = util.UpdateBlackduck(blackDuckClient, currBlackDuck)
-			if err != nil {
-				errors = append(errors, fmt.Errorf("error putting Black Duck '%s' in namespace '%s' into database migration mode due to %+v", blackDuckName, blackDuckNamespace, err))
-				continue
-			}
+		chartRepo := "https://artifactory.internal.synopsys.com/artifactory/bds-hub-helm-snapshot-local/blackduck/blackduck-2020.4.0.tgz"
 
-			log.Infof("successfully modified Black Duck '%s' in namespace '%s' for database migration mode", blackDuckName, blackDuckNamespace)
+		helmValuesMap, err := createBlackDuckCobraHelper.GenerateHelmFlagsFromCobraFlags(cmd.Flags())
+		if err != nil {
+			return err
 		}
-		if len(errors) > 0 {
-			return fmt.Errorf("%v", errors)
+		util.SetHelmValueInMap(helmValuesMap, []string{"status"}, "dbmigrate")
+		// Check Dry Run before deploying any resources
+		err = util.CreateWithHelm3(args[0], namespace, chartRepo, helmValuesMap, kubeConfigPath, true)
+		if err != nil {
+			return fmt.Errorf("failed to create Blackduck resources: %+v", err)
+		}
+
+		// Deploy Polaris-Reporting Resources
```

Comment:
This is just the Create command right? Is it doing the database migrate? 
I think this might depend on if we modify the Helm Chart to have "dbMigrate" as an option for .Values.status

Reply:
Let's ignore db migrate until we decide what to do. If we need to maintain this feature then it first needs to be added to the chart.

Reply:
Sounds good to me